os/board/rtl8721csm/src/component/common/api/wifi/wifi_conf.c

```diff
@@ -63,7 +63,7 @@ extern u8 rtw_get_band_type(void);
  ******************************************************/
 static internal_scan_handler_t scan_result_handler_ptr = {0, 0, 0, RTW_FALSE, 0, 0, 0, 0, 0};
 static internal_join_result_t* join_user_data;
-static unsigned char ap_bssid[6];
+unsigned char ap_bssid[6];

```

Comment:
How about using `ETH_ALEN` instead of `6` to prevent mismatching to line 301 of rtk_netmgr.c?

Reply:
Done.